cmd/opampsupervisor/go.mod

```diff
@@ -23,10 +24,16 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/mitchellh/reflectwalk v1.0.2 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/stretchr/testify v1.8.4 // indirect
 	go.opentelemetry.io/collector/config/configopaque v0.90.2-0.20231201205146-6e2fdc755b34 // indirect
 	go.uber.org/multierr v1.10.0 // indirect
 	golang.org/x/sys v0.14.0 // indirect
 	google.golang.org/protobuf v1.31.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+exclude (
+	github.com/knadh/koanf v1.4.1
+	github.com/knadh/koanf v1.4.2
+	github.com/knadh/koanf v1.4.4
+	github.com/knadh/koanf v1.5.0
```

Comment:
any reason this is being excluded?

Reply:
Good catch, these were just for using Go workspaces locally. I'll remove them.